scripts/components/security/build.sh

```diff
@@ -17,6 +17,7 @@ function usage() {
     echo -e "-p PLATFORM\t[Optional] Platform, ignored."
     echo -e "-a ARCHITECTURE\t[Optional] Build architecture, ignored."
     echo -e "-o OUTPUT\t[Optional] Output path, default is 'artifacts'."
+    echo -e "-d DASHBOARDS\t[Optional] Build OpenSearch Dashboards, default is 'false'."
```

Comment:
What’s this?!

Reply:
This was from the default build.sh in opensearch.

Reply:
https://github.com/opensearch-project/opensearch-build/issues/1634